pkg/config/config.go

```diff
@@ -24,11 +23,10 @@ type ConfigManager struct {
 }
 
 type Config struct {
-	IgnoredVulns                   []IgnoreEntry                        `toml:"IgnoredVulns"`
-	IgnoredPackageVersions         []IgnorePackageVersionEntry          `toml:"IgnoredPackageVersions"`
-	OverridePackageVersionLicenses []OverridePackageVersionLicenseEntry `toml:"OverridePackageVersionLicenses"`
-	LoadPath                       string                               `toml:"LoadPath"`
-	GoVersionOverride              string                               `toml:"GoVersionOverride"`
+	IgnoredVulns      []IgnoreEntry         `toml:"IgnoredVulns"`
+	PackageVersions   []PackageVersionEntry `toml:"PackageVersions"`
```

Comment:
Rename this to `PackageOverrides` and `PackageOverrideEntry`, since version is optional, and we are specifying packages for the scanner to override the interpretation of